.github/workflows/cd_ios.yml

```diff
@@ -43,6 +43,49 @@ jobs:
           cd examples/mobile/ios && bundle install
           bundle exec fastlane build_and_deploy_to_firebase --env enterprise build_number:$GITHUB_RUN_NUMBER
 
+  Build-And-Deploy-To-TestFlight-InternalTest:
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    runs-on: macos-12
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'yarn'
+
+      - name: Install modules
+        run: yarn install
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.1.2
+
+      - name: Assign internal environment
+        env:
+          INTERNAL_RELEASE_TOKEN_URLS: ${{secrets.INTERNAL_RELEASE_TOKEN_URLS}}
+        run: |
+          echo $INTERNAL_RELEASE_TOKEN_URLS > packages/mobile/src/providers/TokenUrl.tsx
```

Comment:
Ok, that's clear. We accept that solution. Please put a short comment about INTERNAL_RELEASE_TOKEN_URLS variable usage so it's clear when checking the yaml code.